### PR TITLE
Mock post badges endpoint

### PR DESCRIPTION
**Pull request by molnarln**
Boti, please review my request!
Thx in advance!

### CODE REVIEW

**Review by Balogh08 — DISMISSED** [review]
+196,000 lines seem to be fine to me.

File: ASP.NET-Core-Webapp/Controllers/BadgeController.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                }
+
+                badge.Levels.Add(new LevelEntity() { Description = "New test skill added", Level = 500, Holders = holdersTest });
+
+                return Created("/badges", new { message = "Success" });
+
+            //}
+            //else
+
+            //    return StatusCode(401, new { error = "Unauthorized" });
```
Comment (szirmi) [inline ASP.NET-Core-Webapp/Controllers/BadgeController.cs]: please always remove unnecessary code

File: ASP.NET-Core-Webapp/Entities/LevelEntity.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+﻿using System.Collections.Generic;
+
+namespace ASP.NET_Core_Webapp.Entities
+{
+    public class LevelEntity
```
Comment (szirmi) [inline ASP.NET-Core-Webapp/Entities/LevelEntity.cs]: be consistent about naming!
if one entity is **Badge**, then this entity should be **Level** (or vice versa)

**Review by szirmi — CHANGES_REQUESTED** [review]
please remove **UnitTestProject1** because it's not part of the application
even packages folder, it shouldn't be in git

**Review by szirmi — APPROVED** [review]

**Pull request merged**